test/integration/targets/consul/tasks/consul_session.yml

```diff
@@ -82,6 +82,51 @@
     that:
       - result is failed
 
+- when: pyopenssl_version.stdout is version('0.15', '>=')
+  block:
+    - name: ensure SSL certificate is checked
+      consul_session:
+        state: info
+        id: '{{ session_id }}'
+        port: 8501
+        scheme: https
+      register: result
+      ignore_errors: True
```

Comment:
```suggestion
      ignore_errors: true
```

Reply:
Why ? [Documentation states](https://docs.ansible.com/ansible/devel/reference_appendices/YAMLSyntax.html):
```
Ansible doesn’t really use these too much, but you can also specify a boolean value (true/false) in several forms:

create_key: yes
needs_agent: no
knows_oop: True
likes_emacs: TRUE
uses_cvs: false

```

Reply:
Mostly for style: you also used `false` in the same patch and not `False`. I've now looked at the whole file, everywhere else `True` and `False` is used. So maybe change `false` to `False`. Anyway, I was more thinking about docs, where it was said somewhere (don't know right now) that we should always use `true`/`false` or `yes`/`no`.

Anyway, choose whatever boolean form you want, but it would be nice if its a bit more consistent here. If you want to keep the current version, tell me, then I'll just merge :)

Reply:
Updated: `True`/`False` used everywhere.